src/module/Taxonomy/src/Taxonomy/Controller/TermController.php

```diff
@@ -159,6 +146,19 @@ public function batchCopyAction()
         return $view;
     }
 
+    private function batchElementsArray()
+    {
+        $term     = $this->getTaxonomyManager()->getTerm($this->params('term'));
+        $elements = $term->getAssociated('entities');
+        $options  = [];
+        foreach ($elements as $element) {
```

Comment:
Could you instead use [`Uuid\Filter\NotTrashedCollectionFilter`](https://github.com/serlo-org/athene2/blob/master/src/module/Uuid/src/Uuid/Filter/NotTrashedCollectionFilter.php)?